README.md

```diff
@@ -92,6 +91,71 @@ name: Pronto
         git fetch --no-tags --prune origin
 ```
 
+# ESLint
+
+Because of big variety of different plugins and options for ESLint configuration, you need install them manually before running pronto.
+
+Example minimal setup:
+
+```
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: borales/actions-yarn@v2.0.0
```

Comment:
Does node come pre-installed on the `ubuntu-latest` VM? I didn't realize that.

Reply:
Question: we've just been using `yarn install --ignore-optional --ignore-scripts --frozen-lockfile --non-interactive`. Do you think there's an advantage to using this action vs. letting the user provide their own install command and just having an example here?

Reply:
To be honest - it is faster what I found to do a job, didn't give a lot of thoughts here. 
I guess it will work for most of the users, but I am open to your suggestion.